bindings/na/zoned/zoned.cpp

```diff
@@ -39,6 +39,16 @@ PYBIND11_MODULE(MQT_QMAP_MODULE_NAME, m, py::mod_gil_not_used()) {
                           &na::zoned::Architecture::fromJSONFile, "filename"_a);
   architecture.def_static("from_json_string",
                           &na::zoned::Architecture::fromJSONString, "json"_a);
+  architecture.def(
+      "to_namachine_file",
+      [](na::zoned::Architecture& self, const std::string& filename) -> void {
+        self.exportNAVizMachine(filename);
+      },
+      "filename"_a);
+  architecture.def("to_namachine_string",
+                   [](na::zoned::Architecture& self) -> std::string {
+                     return self.exportNAVizMachine();
+                   });
```

Comment:
No need to use lambdas here. You can simply export the member function.

Reply:
Ah. Missed that. For the sake of simplicity, let's just leave this as is.

Differentiating the functions would probably make sense, generally. And I think there's an overload_cast or similar in pybind that would also allow to choose the right function.